Tasks/SqlAzureDacpacDeploymentV1/task.json

```diff
@@ -17,7 +17,7 @@
     "version": {
         "Major": 1,
         "Minor": 156,
-        "Patch": 0
+        "Patch": 1
```

Comment:
Why this change is required?

Reply:
Since I made some changes in Common, it was required to bump up version of some of these tasks, so updated patch version